CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - Add error and sample rate fields to the replay event parser. ([#1745](https://github.com/getsentry/relay/pull/1745))
 - Add `instruction_addr_adjustment` field to `RawStacktrace`. ([#1716](https://github.com/getsentry/relay/pull/1716))
 - Add SSL support to `relay-redis` crate. It is possible to use `rediss` scheme to connnect to Redis cluster using TLS. ([#1772](https://github.com/getsentry/relay/pull/1772))
+- Add count transactions toward root project. ([#1734](https://github.com/getsentry/relay/pull/1734))
```

Comment:
Please, move this to `Unreleased` section.

Reply:
@olksdr done.